test/asyncapi_test.go

```diff
@@ -40,6 +40,29 @@ func (s *CLITestSuite) TestAsyncApiExport() {
 			s.Error(nil, "spec generated does not match the template output file")
 		}
 	}
+	resetConfiguration(s.T(), false)
+}
 
+func (s *CLITestSuite) TestAsyncApiImport() {
+	tests := []CLITest{
+		//Input file not specified
+		{args: "asyncapi import", exitCode: 1},
```

Comment:
You should have golden files for each command. The names of these files can serve as documentation, so you don't need comments.
```suggestion
		{args: "asyncapi import", fixture: "asyncapi/err-no-file.golden", exitCode: 1},
```